### PR TITLE
[4.0][Admin menu] Allow multiple menus including expanded ones in sidebar

### PR DESCRIPTION
**Pull request by infograf768**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/25953

### Summary of Changes
Thanks @Fedik
The admin-menu js was only loading the metismenu event on the first occurence of a menu.
This PR also creates different ids per menu

### Steps to reproduce the issue
Create a custom admin menu with no presets (we have other issues with presets and I will create another issue for that.)
Create a menu module for that menu in the "menu" position.
Make sure the module is placed AFTER the default Admin Menu module in the Manager.

Create a Heading menu item in that custom admin menu.
Create any other menu item as child of that Heading menu item.

The result is that we have both the default Admin Menu and this custom one in the menu side bar. Below, the custom admin menu starts with Global Config.

Screen Shot 2019-08-21 at 10 31 37

### Before patch
The items with arrows in the second menu do not expand

### After patch
Issue solved

### CODE REVIEW

**Comment by infograf768** [comment]
Need help for hound... @Fedik @wilsonge

File: build/media_source/mod_menu/js/admin-menu.es6.js [hunk 1]
```diff
@@ -21,8 +21,11 @@
     }
   };
 
-  // eslint-disable-next-line no-new
-  new window.MetisMenu('#menu');
+  const allMenus = document.querySelectorAll('ul.main-nav');
+  allMenus.forEach((menu) => {
+    // eslint-disable-next-line no-new
+    new MetisMenu(menu);
```
[the remark this answers is not in the document]
Comment (Fedik) [reply build/media_source/mod_menu/js/admin-menu.es6.js:27]: try `// eslint-disable-next-line no-new, no-undef `

File: build/media_source/mod_menu/js/admin-menu.es6.js [hunk 2]
```diff
@@ -22,7 +22,10 @@
   };
 
   // eslint-disable-next-line no-new
-  new window.MetisMenu('#menu');
+  const allMenus = document.querySelectorAll('ul.main-nav');
+  allMenus.forEach(menu => {
```
[the remark this answers is not in the document]
Comment (Fedik) [reply build/media_source/mod_menu/js/admin-menu.es6.js]: here `allMenus.forEach((menu) => {`

**Comment by infograf768** [comment]
Shall I use `/* eslint no-undef: "off"*/` at top of file, or is there another way to make hound happy?

**Comment by infograf768** [comment]
At last, hound happy.... GRRRR

File: build/media_source/mod_menu/js/admin-menu.es6.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (infograf768) [reply build/media_source/mod_menu/js/admin-menu.es6.js]: this was solved

**Comment by Fedik** [comment]
I have tested this item :white_check_mark: successfully on 199e91631b182165e22c01391a9eecc9b867833f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25995">issues.joomla.org/tracker/joomla-cms/25995</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 199e91631b182165e22c01391a9eecc9b867833f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25995">issues.joomla.org/tracker/joomla-cms/25995</a>.</sub>

**Comment by ghost** [comment]
Status "Ready To Commit".

**Comment by HLeithner** [comment]
Thanks for this menu fix.

**Pull request merged**